shared/getting-started/deployingCode/nodejs.md

```diff
@@ -1,5 +1,6 @@
-A nice first project to get started is a simple [Express.js][expressjs] web server which, will serve a static page on port `:80`. All the project source code can be found [here on GitHub][simple-server-node-link]. Download a zipped file of the project [here][github-download], and when downloaded, unzip the file and open a terminal in the root of the project directory.
+A nice first project to get started is a simple [Express.js][expressjs] web server which, will serve a static page on port `:80`. All the project source code can be found [here on GitHub][simple-server-node-link]. Download a zipped file of the project [here][github-download-js], and when downloaded, unzip the file and open a terminal in the root of the project directory. For a TypeScript version of the project please go [here][github-download-ts].
 
 [simple-server-node-link]:{{ $links.githubLabs }}/simple-server-node
-[github-download]:{{ $links.githubLabs }}/simple-server-node/archive/master.zip
+[github-download-js]:{{ $links.githubLabs }}/simple-server-node/archive/master.zip
```

Comment:
```suggestion
[github-download-js]:{{ $links.githubExamples }}/balena-node-hello-world/archive/master.zip
```
Please test to confirm, but I understand that repo was renamed in _The Great Renaming Exercise_ last year, and it makes sense for the docs to follow suit even if the old names still redirect to the new names.

Reply:
Same as above, done.